src/index.ts

```diff
@@ -1,3 +1,51 @@
-// TODO: write you code here
+import { toError } from "fp-ts/lib/Either";
+import { tryCatch } from "fp-ts/lib/TaskEither";
+import * as fs from "fs";
+import newApp from "./app";
+import EmailService from "./services/emailService";
+import { getRequiredEnvVar } from "./utils/environment";
+import { log } from "./utils/logger";
 
-process.stdout.write("Hello world");
+// read package.json to print some info
+const packageJson = JSON.parse(fs.readFileSync("./package.json").toString());
+
+const emailService = new EmailService(
+  {
+    auth: {
```

Comment:
another option would be to have a Configuration io-tstypes and call `Config.decode(process.env)`